package.json

```diff
@@ -32,7 +32,8 @@
     "react-router-dom": "6.9.0",
     "react-scripts": "5.0.1",
     "web-vitals": "3.3.1",
-    "yup": "1.1.1"
+    "yup": "1.1.1",
+    "zustand": "^4.3.8"
```

Comment:
Change this to "zustand": "4.3.8" (better to have a stable version)

Reply:
Okay